integration/tests/cook/test_cli.py

```diff
@@ -1606,6 +1614,10 @@ def test_cat_bogus_uuid(self):
         self.assertEqual(1, cp.returncode, cp.stdout)
         self.assertIn('No matching data found', cli.decode(cp.stderr))
 
+    @unittest.skipIf(util.has_ephemeral_hosts(), 'If the cluster under test has ephemeral hosts, then it is generally '
+                                                 'a bad idea to use HOSTNAME EQUALS constraints, because it can cause '
+                                                 'the process responsible for launching hosts to launch hosts that '
+                                                 'never get used')
```

Comment:
Can we have this string in a constant somewhere and refer to the constant in the annotation?

Reply:
Done